### DISCUSSION TITLE
Hi I’m Matteo!

### DISCUSSION THREAD

**Discussion by CubeForm00** [smartcontractkit/full-blockchain-solidity-course-py#1519] [General]
Hi,I'm Matteo.
I'm totally new to programming and web3 and I'm so excited to jump in this wonderful world!

**Comment by cromewar** [comment]
Hello @CubeForm00 
Welcome to blockchain! Feel free to make any question while your a re taking the course.
And also, please join the discord community (chainlink server).
Cheers.

**Comment by Ondion** [comment]
Hello! 👍

**Comment by BasedAltcoin** [comment]
Also new to coding and web3, looking for a career change into blockchain development!
